<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayerNumber</class>
 <widget class="QDialog" name="PlayerNumber">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>640</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>960</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Love Letter</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/ressources/game_icon.png</normalon>
   </iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">/*
background-color: rgb(81, 60, 117);
*/</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>960</width>
     <height>75</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>960</width>
     <height>75</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>960</width>
     <height>75</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>28</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>SELECT THE NUMBER OF PLAYERS </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>881</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="twoPlayersButton">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>100</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: rgb(255,229,232);
	color: rgb(227,91,120);
}

QPushButton:checked{
	background-color: rgb(227,91,120);
	color: rgb(255,229,232);
}</string>
      </property>
      <property name="text">
       <string> 2 PLAYERS</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="autoRepeat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="threePlayersButton">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>100</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: rgb(255,229,232);
	color: rgb(227,91,120);
}

QPushButton:checked{
	background-color: rgb(227,91,120);
	color: rgb(255,229,232);
}</string>
      </property>
      <property name="text">
       <string>3 PLAYERS</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="fourPlayersButton">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>100</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: rgb(255,229,232);
	color: rgb(227,91,120);
}

QPushButton:checked{
	background-color: rgb(227,91,120);
	color: rgb(255,229,232);
}</string>
      </property>
      <property name="text">
       <string>4 PLAYERS</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>320</y>
     <width>960</width>
     <height>75</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>960</width>
     <height>75</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>960</width>
     <height>75</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
</string>
   </property>
   <property name="text">
    <string>SELECT MODE</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>430</y>
     <width>631</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="computerButton">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>100</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: rgb(255,229,232);
	color: rgb(227,91,120);
}

QPushButton:disabled{
	background-color: rgb(194,165,171);
	color: rgb(79,63,66);
}</string>
      </property>
      <property name="text">
       <string>COMPUTER </string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="onlineButton">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>100</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: rgb(255,229,232);
	color: rgb(227,91,120);
}

QPushButton:disabled{
	background-color: rgb(194,165,171);
	color: rgb(79,63,66);
}</string>
      </property>
      <property name="text">
       <string>ONLINE</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
